flow-typed/box-ui-elements.js

```diff
@@ -892,3 +892,35 @@ type AdditionalVersionInfo = {
     currentVersionId?: ?string,
     updateVersionToCurrent: () => void,
 };
+
+type MetadataQeuryResponseEntryEnterprise = {
```

Comment:
We're trying to eliminate global types. Can these types live in something like `src/common/types/metadataQueries.js`?

Reply:
Will make that change.